fplus-lib/src/core/application/file.rs

```diff
@@ -428,8 +428,19 @@ pub struct Verifier {
     pub signing_address: String,
     #[serde(rename = "Created At")]
     pub created_at: String,
+    #[serde(rename = "Message CIDs")]
+    pub message_cids: GrantDataCapCids,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct GrantDataCapCids {
     #[serde(rename = "Message CID")]
-    pub message_cid: String,
+    pub message_cids: String,
```

Comment:
```suggestion
    pub message_cid: String,
```

It's still just one CID, right?